lib/git-shell-out-strategy.js

```diff
@@ -487,12 +508,26 @@ export default class GitShellOutStrategy {
       // add untracked file
       const absPath = path.join(this.workingDir, filePath);
       const executable = await isFileExecutable(absPath);
+      const symlink = await isFileSymlink(absPath);
```

Comment:
wondering if this will return the correct thing on Windows (e.g. could Git be configured to recognize them or have a stored symlink filemode but the fs helpers return `false`)